src/mococrw/distinguished_name.h

```diff
@@ -85,6 +85,14 @@ class DistinguishedName
     {
         return _getAttributeByNIDAsString(openssl::ASN1_NID::GivenName);
     }
+
+    /**
```

Comment:
I agree, especially since the other functions are also undocumented. I raised this inconsistency, however I was explicitly asked to add the comments for the doxygen. It is also part of the acceptance criteria.